docs/features/pause.md

```diff
@@ -0,0 +1,43 @@
+# Pause KCC
+
+> `Feature State`: KCC version v1.114+ (alpha)
+
+KCC can be configured to pause actuation of resouces on the cloud provider
+(GCP). K8s objects continue to be reconciled with the api server but any
+interaction with the cloud provider should be paused. This can be helpful for
+debugging purposes or to have a hot standby.
+
+## Pausing
+
+We extended the API definitions for the ConfigConnector and ConfigConnectorContext resources to
+support a new field, `spec.actuationMode`. The field's current supported values are `Reconciling`
+and `Paused` with `Reconciling` being the default for backwards compatiblity.
+As such, KCC can be "paused" both globablly and on a per namespace level if running in namesapce mode
```

Comment:
Typo in globally.  And probably `per-namespace` (with a hyphen).  Also typo in namespace :-)